kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerConfiguration.java

```diff
@@ -163,21 +145,12 @@ public String getContainerLoggingType() {
     }
 
     /**
-     * Returns the Registry credentials
+     * @since 2.4 Returns the {@link ImageConfiguration} object
      *
      * @return
      */
```

Comment:
same here

Reply:
Also for since annotations, please follow the schema used for the other APIs

Reply:
/**
      * Returns the {@link ImageConfiguration} object
      *
      * @return
      * `@since 2.4` 
      */